designernews/src/main/java/io/plaidapp/designernews/ui/story/StoryViewModel.kt

```
@@ -73,4 +83,26 @@ class StoryViewModel(
         parentJob.cancel()
         super.onCleared()
     }
+
+    private fun getComments() = launch(contextProvider.io, parent = parentJob) {
+        story.links?.let {
+            val result = commentsWithRepliesAndUsers(it.comments)
+            if (result is Result.Success) {
```

Optionally add comment here that Result.Error is ignored